pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -216,6 +216,26 @@ func (s *Launcher) GetDomain(args *cmdclient.Args, reply *cmdclient.Reply) error
 	return nil
 }
 
+func (s *Launcher) GetDomainStats(args *cmdclient.Args, reply *cmdclient.Reply) error {
```

Comment:
Why is this function returning error? This function is returning everytime nil.

Reply:
I anticipated an error handling I never added - and turns out, is not really needed. I'll drop that.

Reply:
Actually I'm keeping this to be consistent with other functions in this module.